code/messaging/build.gradle

```diff
@@ -90,7 +115,7 @@ def isReleaseBuild() {
     return hasProperty('release')
 }
 
-version = isReleaseBuild() ? rootProject.moduleVersion : rootProject.moduleVersion+"-SNAPSHOT"
+version = isReleaseBuild() ? rootProject.moduleVersion : rootProject.moduleVersion + "-SNAPSHOT"
```

Comment:
nit: since moduleversion is part of gradle properties, we can access it directly i.e `moduleversion`, without using rootProject to access it.